### DISCUSSION TITLE
Authoring plugins written in Rust?

### DISCUSSION THREAD

**Discussion by teymour-aldridge** [swc-project/swc#1335] [Q&A]
Is there any documentation for writing bundler plugins in Rust?

(I'm trying to write a plugin which will bundle together a Javascript module and produce a set of Rust + WebAssembly bindings to it).

Best,
Teymour

**Comment by kdy1** [comment]
At the moment, there's no api for plugin.
It's because I couldn't find proper design without actual usecases.

Can you describe more about it to help me designing apis?
